common/djangoapps/student/helpers.py

```diff
@@ -2,8 +2,9 @@
 from datetime import datetime
 import urllib
 
-from pytz import UTC
 from django.core.urlresolvers import reverse, NoReverseMatch
+from provider.oauth2.models import AccessToken, RefreshToken
```

Comment:
Remember to destroy tokens for both django-oauth2-provider *and* django-oauth-toolkit.